### ISSUE TITLE
rt3290sta-2.6.0.0 is missing

### ISSUE THREAD

**Issue by m-alex-n** [Borwe/rt3290-linux-drivers#4]
when running sudo dkms install -m rt3290sta -v 2.6.0.0 --force  i get 
![Screenshot from 2020-08-09 20-55-36](https://user-images.githubusercontent.com/62503960/89739303-4cf8c280-da88-11ea-92f3-16b0d9c98b5a.png)

**Comment by AkibAzmain** [comment]
What do you mean?

**Comment by Borwe** [comment]
Yeah there is missing **Build.include** file that is required by **Makefile.build** @ **/usr/src/linux-headers-5.4.0-42-generic/tools/build/** in that kernel 5.4.0.42 specifically on Ubuntu, (I haven't tried on other distros), PR is welcomed for any solutions.

**Comment by christiankozalla** [comment]
I'm running Ubuntu 20.04 and get the same error as described above. Is the problem easy to fix for myself? I have very little experience with linux and c... Thanks!
